src/components/button-toggle/button-toggle.config.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export const BUTTON_TOGGLE_ICON_SIZES = ["small", "large"];
```

Comment:
why do we need a separate file for that?

Reply:
We don't need as it is used only in one file :) Good spot